src/components/Wallet/WalletControl.jsx

```diff
@@ -81,7 +78,7 @@ class WalletControl extends React.Component {
           />
         );
       if (walletMode === WALLET_MODES.VIEW)
-        return <WalletView addNode={addNode} updateNode={updateNode} />;
```

Comment:
`WalletView` was a redundant component that was just rendering the `NodeSet` component. I deleted the component and just used `NodeSet` directly. Seemed to work fine.